modules/fedimint-wallet/src/tweakable.rs

```diff
@@ -21,7 +21,8 @@ impl Tweakable for secp256k1::PublicKey {
         let tweak = Hmac::from_engine(hasher).into_inner();
 
         let mut key = *self;
-        key.add_exp_assign(secp, &tweak[..])
+        // TODO: is this supposed to be be or le?
+        key.add_exp_tweak(secp, &Scalar::from_be_bytes(tweak).expect("can't fail"))
```

Comment:
le or be?

Reply:
Whatever we use otherwise, it doesn't make any difference afaiu.